config/rbac/role.yaml

```diff
@@ -49,7 +49,9 @@ rules:
   resources:
   - machineconfigs
   verbs:
-  - '*'
+  - get
```

Comment:
it will not work, because our operator should created machine configs

Reply:
ah, right. Good point. Do we also need delete permissions, though?

Reply:
I think we need all permissions, like create, update, patch, watch, delete, list and get